gradle/libs.versions.toml

```diff
@@ -1,53 +1,46 @@
 [versions]
-android-plugin = "8.1.1"
-classgraph = "4.8.170"
+android-plugin = "8.4.1"
+classgraph = "4.8.172"
 dataloader = "3.3.0"
-federation = "4.4.1"
+federation = "5.0.0"
 graphql-java = "0.0.0-2024-04-23-expedia-branch-1"
-graalvm = "0.10.1"
-jackson = "2.15.4"
+graalvm = "0.10.2"
+jackson = "2.17.1"
 # kotlin version has to match the compile-testing compiler version
-kotlin = "1.8.22"
-# TODO kotlin 1.9 upgrade -> kotlinx-benchaamrk 0.4.9+ uses kotlin 1.9
-kotlinx-benchmark = "0.4.8"
-# TODO kotlin 1.9 upgrade -> kotlinx-coroutines 1.8+ uses kotlin 1.9
-kotlinx-coroutines = "1.7.3"
-# TODO kotlin 1.9 upgrade -> kotlinx-serialization 1.6.0+ uses kotlin 1.9
-kotlinx-serialization = "1.5.1"
-ktor = "2.3.10"
-maven-plugin-annotation = "3.12.0"
+kotlin = "1.9.24"
+kotlinx-benchmark = "0.4.10"
+kotlinx-coroutines = "1.8.1"
+# TODO kotlin 1.9 upgrade: fix GraphQLTestUtils and GenerateKotlinxClientIT
+kotlinx-serialization = "1.6.3"
+ktor = "2.3.11"
+maven-plugin-annotation = "3.13.0"
 maven-plugin-api = "3.9.6"
 maven-project = "2.2.1"
-# TODO kotlin 1.9 upgrade -> kotlinxpoet 1.15+ uses kotlin 1.9
-poet = "1.14.2"
+poet = "1.17.0"
 ## reactor and spring versions should be the same as defined in spring-boot-dependencies
-reactor-core = "3.5.15"
+reactor-core = "3.6.6"
 reactor-extensions = "1.2.2"
-slf4j = "2.0.12"
-spring = "6.0.18"
-# TODO kotlin 1.9 upgrade -> spring boot 3.2+ uses kotlin 1.9
-spring-boot = "3.1.10"
+slf4j = "2.0.13"
+spring = "6.1.8"
+spring-boot = "3.3.0"
 
 # test dependencies
-compile-testing = "0.3.2"
-icu = "74.2"
+compile-testing = "0.4.1"
+icu = "75.1"
 junit = "5.10.2"
-logback = "1.5.4"
-# TODO kotlin 1.9 upgrade -> mockk 1.13.8+ uses kotlin 1.9
-mockk = "1.13.7"
+logback = "1.5.6"
+mockk = "1.13.11"
 rxjava = "3.1.8"
-wiremock = "3.5.2"
+wiremock = "3.6.0"
 
 # plugins
-# TODO kotlin 1.9 upgrade -> detekt 1.23.1+ uses kotlin 1.9
-detekt = "1.23.0"
-dokka = "1.8.20"
+detekt = "1.23.6"
+dokka = "1.9.20"
 jacoco = "0.8.12"
-# klint gradle plugin breaks with 0.46.x+
-ktlint-core = "0.45.2"
-ktlint-plugin = "10.3.0"
+ktlint-core = "1.2.1"
+ktlint-plugin = "12.1.1"
 maven-plugin-development = "0.4.3"
-nexus-publish-plugin = "1.3.0"
+nexus-publish-plugin = "2.0.0"
```

Comment:
I believe this change requires change to release workflow -> i.e. task was renamed from `closeAndReleaseSonatypeStagingRepository` to `closeAndReleaseStagingRepositories`

Reply:
Your workflow uses `closeAndReleaseSonatypeStagingRepository` and succeeded, the breaking change affected only `closeAndReleaseStagingRepository`:
https://github.com/gradle-nexus/publish-plugin/releases/tag/v2.0.0